tool/tsh/tsh.go

```diff
@@ -2117,3 +2122,21 @@ func readClusterFlag(cf *CLIConf, fn envGetter) {
 // envGetter is used to read in the environment. In production "os.Getenv"
 // is used.
 type envGetter func(string) string
+
+func handleUnimplementedError(ctx context.Context, cf CLIConf) error {
+	const (
+		errMsgFormat         = "This server does not implement this feature yet. Likely the client version you are using is newer than the server. The server version: %v, the client version: %v. Please upgrade the server or downgrade the client."
```

Comment:
> Not sure we should recommend downgrading the client, that seems unlikely to help with what they are actually trying to do.

done
> Should we wrap the original error rather than returning a new one, to include the original message? It would be uglier but IMO way more helpful if a user runs into this and needs to debug.

done, I have wrapped parent error message, it will look like below:
```
ERROR: This server does not implement this feature yet. Likely the client version you are using is newer than the server. The server version: 5.2.1, the client version: 7.0.0-dev. Please upgrade the server or downgrade the client.
unknown method GetDatabaseServers for service proto.AuthService
```

> Also just a nit: it's my understanding that we usually don't capitalize the first letter of error messages because they are likely to be preceded by something.

It will look a bit weird for me where only first error sentence is not capitalized.

Reply:
>>Not sure we should recommend downgrading the client, that seems unlikely to help with what they are actually trying to do.

>done

Sorry, seems like it is still there? The rest looks good to me

Reply:
🙃 sorry for that, updated.